pkg/specgen/specgen.go

```diff
@@ -307,6 +307,15 @@ type ContainerSecurityConfig struct {
 	Umask string `json:"umask,omitempty"`
 	// ProcOpts are the options used for the proc mount.
 	ProcOpts []string `json:"procfs_opts,omitempty"`
+	// Mask is the path we want to mask in the container. This masks the paths
+	// given in addition to the default list. Multiple paths can be specified using ":"
+	// e.g /foo/bar:/hellow/world
+	// Optional
+	Mask string `json:"mask,omitempty"`
```

Comment:
This and Unmask should be `[]string` - we don't want to do string parsing in the backend, let the client send us a list of paths already ready to use.

Reply:
So you are saying if the user wants to mask multiple paths, they will have to do 
```
podman run --security-opt mask=/path/1 --security-opt mask=/path/2 ... ?
```
That seems a bit tedious on the user end, what is the issue with supporting multiple paths separated by a colon? This will definitely be more user friendly where they have many paths that they want to mask or unmask.

Reply:
No. I am saying that we need to do the parsing in `cmd/podman`. Once we get to Specgen, the input should already have been separated into a `[]string`.

Reply:
One of the goals of Specgen is to remove parsing from the equation. Everything passed into Specgen should, to the greatest extent possible, be in its simplest possible form. Parsing should be done on the client side, in `cmd/podman`. The server side, `pkg/specgen`, should receive an already-parsed, unambiguous form.

Reply:
oh okay, got it. Will fix.

Reply:
Fixed